docs/products/access/release-notes/index.md

```diff
@@ -8,6 +8,21 @@ The releases are backward compatible. However, we will extend the API contract (
 
 In the release notes, we mention new features and bug fixes. If anything is unclear, feel free to contact [OneWelcome Support](https://support.onewelcome.com).
 
+## Release date 2023-10-30
+
+### Features
+* We now support more [`acr` values](../topics/oidc/overview/scopes-and-claims.md#acr) for non-CIM-type IdPs. These `acr_values` are also exposed in the Discovery Endpoint.
```

Comment:
`non-CIM-type` idps is really technical. Can we rephrase that to something more easily understood?

Could we link to `../api-reference/oidc/discovery.md` for the discovery endpoint. (and also update that md file? hasn't been touched since 2021)